### PR TITLE
`from url` and `from eml`

### PR DESCRIPTION
**Pull request by luccasmmg**
```
〉"Date: Sun, 1 Apr 2012 14:25:25 -0600
From: file@fyicenter.com
Subject: Welcome
To: someone@somewhere.com

Dear Friend,

Welcome to file.fyicenter.com!

Sincerely,
FYIcenter.com Team" | from eml
╭─────────┬──────────────────────────────────────────╮
│ Body    │ Dear Friend,                             │
│         │ Welcome to file.fyicenter.com!           │
│         │ Sinc                                     │
│ Date    │ Sun, 1 Apr 2012 14:25:25 -0600           │
│ To      │ {Name: , Address: someone@somewhere.com} │
│ Subject │ Welcome                                  │
│ From    │ {Name: , Address: file@fyicenter.com}    │
╰─────────┴──────────────────────────────────────────╯

```

```
〉"Date: Sun, 1 Apr 2012 14:25:25 -0600
From: file@fyicenter.com
Subject: Welcome
To: someone@somewhere.com

Dear Friend,

Welcome to file.fyicenter.com!

Sincerely,
FYIcenter.com Team" | from eml -b 1 | get Body
D
```

```
〉"bread=baguette&cheese=comt%C3%A9&meat=ham&fat=butter" | from url
╭────────┬──────────╮
│ bread  │ baguette │
│ cheese │ comté    │
│ meat   │ ham      │
│ fat    │ butter   │
╰────────┴──────────╯
```

### CODE REVIEW

File: crates/nu-command/src/formats/from/eml.rs [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                span: head,
+            },
+        );
+    }
+
+    Ok(PipelineData::Value(Value::from(collected)))
+}
+
+#[cfg(test)]
+mod tests {
```
Comment (onthebridgetonowhere) [inline crates/nu-command/src/formats/from/eml.rs:176]: Could you add some tests here? That would be great!

**Comment by jntrnr** [comment]
Oh awesome. Good to see these.

A test to eml could be nice, agreed. Maybe there's a tiny email we could use?

**Comment by luccasmmg** [comment]
I tried adding a test, but there is a problem, `Value::Record` implementation of `PartialOrd` requires the columns and vals to be in the same order, i can change this behaviour to first try to sort the values and column names and only then compare the Records, but i wanted to known first with you if this is desired or not. This problem arises from the fact that the test suite uses partial_cmp to check if the output is expected. Or if you know how to make the command always have the output in the same order, i would also be happy to hear that.

**Comment by luccasmmg** [comment]
@jntrnr to make the tests work i had to add a case for `PartialOrd` where two `Value::Nothing` are always equal, i assume that there is no problem with that, right?

**Comment by jntrnr** [comment]
@luccasmmg yeah, two Nothings should be equal. Sounds good

**Comment by luccasmmg** [comment]
@jntrnr did you saw that i added the tests?

**Comment by jntrnr** [comment]
Ah I missed it. Looks good

**Pull request merged**